Source/WebInspectorUI/UserInterface/External/CSSDocumentation/CSSDocumentation-overrides.json

```diff
@@ -4,6 +4,6 @@
     },
     "-webkit-appearance": {
         "description": "Changes the appearance of buttons and other controls to resemble native controls.",
-        "syntax": "none | button | caret | checkbox | default-button | inner-spin-button | listbox | listitem | media-play-button | media-slider | media-sliderthumb | media-volume-slider | media-volume-sliderthumb | menulist | menulist-button | meter | progress-bar | progress-bar-value | push-button | radio | searchfield | searchfield-cancel-button | searchfield-decoration | searchfield-results-button | searchfield-results-decoration | slider-horizontal | slider-vertical | sliderthumb-horizontal | sliderthumb-vertical | square-button | textarea | textfield | -apple-pay-button"
+        "syntax": "none | button | checkbox | default-button | listbox | media-play-button | media-slider | media-sliderthumb | media-volume-slider | media-volume-sliderthumb | menulist | menulist-button | meter | progress-bar | push-button | radio | searchfield | slider-horizontal | slider-vertical | square-button | textarea | textfield | -apple-pay-button"
```

Comment:
you'll also need to make the same change to `Source/WebInspectorUI/UserInterface/External/CSSDocumentation/CSSDocumentation.js`

do we want to change the values for the `appearance` property too?  or are many of these things nonstandard?

Reply:
`appearance` seems to have `"none | auto | textfield | menulist-button | <compat-auto>"` as syntax (taken from MDN I'm guessing), which is valid as well, and doesn't have non-standard values